src/services/PostmanSyncService.ts

```diff
@@ -45,6 +51,15 @@ export class PostmanSyncService {
 
     this.portmanCollection = portmanCollection
     this.collectionName = collectionName || (portmanCollection?.info?.name as string)
+
+    this.postmanFastSync = postmanFastSync || false
```

Comment:
should use `??` instead of `||` when checking for bools that might be undefined

Reply:
done